Sources/CocoaTextField/CocoaTextField.swift

```diff
@@ -73,6 +73,14 @@ public class CocoaTextField: UITextField {
         didSet { errorLabel.textColor = errorColor }
     }
     
+    /**
+    * Sets text field's isSecureTextEntry property
+    */
+    @IBInspectable
```

Comment:
won't it work by just overriding `isSecureTextEntry` and setting it `@IBInspectable` ?